doc/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Introduction
+author: Frederik Rietdijk
+date: 2015-11-25
+---
+
+The Nix Packages collection (Nixpkgs) is a set of over 6,500 packages for the
```

Comment:
I'm not sure what to count. There are also a few thousand TeX packages now (hidden to nix-env). But the order would be higher than what's written here IMHO, i.e. *?0 000*.